apps/web/components/SearchInput/SearchInput.tsx

```diff
@@ -135,10 +138,10 @@ const useSearch = (
         })
 
       // the api only completes single terms get only single terms
-      const indexOfLastSpace = term.lastIndexOf(' ')
+      const indexOfLastSpace = term?.lastIndexOf(' ')
       const hasSpace = indexOfLastSpace !== -1
```

Comment:
Perhaps we should take into consideration that indexOfLastSpace could be undefined in this check